app/Http/Controllers/ArticleCategoryController.php

```diff
@@ -12,54 +17,58 @@ class ArticleCategoryController extends Controller
      */
     public function index()
     {
-        //
+        $query=ArticleCategory::Query();
+        $request=Request();
+        if($search=$request->Catergory){
+            $query->where("articleCategoryName","LIKE","%$search%");
+        }
+        $Categories=$query->get();
```

Comment:
Personally, I don't like to name variable starting with capital letter.

PSR standards says you have to follow one convention over a level, so just make your variable `camelCase` to just unify our work.